setup.py

```diff
@@ -17,7 +17,7 @@
         'Intended Audience :: Developers',
         'Intended Audience :: Science/Research',
         'License :: OSI Approved :: MIT License',
-        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
     ],
     packages=['ceci', 'ceci.sites', 'ceci_example'],
```

Comment:
This is actually a bit of a pain to do separately, because of how ceci searches for modules (it expects packages to be run from the top level).  This is kind of a bug.  I've removed it from the installer though.